README.md

```diff
@@ -1,3 +1,7 @@
+# Deprecation warning
+
+**This project is being deprecated. It will no longer receive any updates, including security patches, so we recommend to stop using it as a dependency. The repository will be archived.**
```

Comment:
```suggestion
**This project is being deprecated. It will no longer receive any updates, including security patches, so we recommend to stop using it as a dependency (or to use it at your own risk). The repository will be archived.**
```